.github/workflows/android-perf.yml

```diff
@@ -33,7 +33,7 @@ jobs:
             procPerformanceThresholdSec: 1.0
           - device: 32bit-android
             initPerformanceThresholdSec: 1.0
-            procPerformanceThresholdSec: 1.0
+            procPerformanceThresholdSec: 7.0
```

Comment:
how did this from 1 to 7?

Reply:
it was never 1, I put that arbitrarily, then ran the AppCenter test to see how long it takes and updated it after

Reply:
oh i see